src/main/java/com/team1816/season/Robot.java

```diff
@@ -531,6 +543,11 @@ public void disabledPeriodic() {
                         autoModeManager.getSelectedAuto().getCurrentTrajectory()
                     );
             }
+
+            // check if demo mode speed multiplier changed
+            if (ControlBoardBrige.getInstance().isDemoMode()) {
+                controlBoard.update();
```

Comment:
Should be a better way to do this than polling in disabledPeriodic. What was the issue with the entry listener?

Reply:
Yes this should be event driven.